olive/model/handler/onnx.py

```diff
@@ -32,7 +32,13 @@ class ONNXModelHandler(OliveModelHandler, OnnxEpValidateMixin, OnnxGraphMixin):
     the mixin class OnnxGraphMixin is used to support onnx graph operations.
     """
 
-    json_config_keys: Tuple[str, ...] = ("onnx_file_name", "inference_settings", "use_ort_extensions")
+    json_config_keys: Tuple[str, ...] = (
+        "onnx_file_name",
+        "inference_settings",
+        "use_ort_extensions",
+        "external_initializers_name",
```

Comment:
>external_initializers_name [](http://example.com/codeflow?start=9&length=26)

external_initializers_file_name?

Reply:
sure, will change to `external_initializers_file_name` and `constant_inputs_file_name` to make it clear they are file names.

Reply:
done.